src/cljs/org/broadinstitute/firecloud_ui/page/workspace/data/tab.cljs

```diff
@@ -59,28 +59,28 @@
    :render
    (fn [{:keys [state props]}]
      [comps/OKCancelForm
-      {:header "Import Data"
-       :show-cancel? false :ok-button {:text "Done" :onClick modal/pop-modal}
+      {:header "Import Data" :show-cancel? true :cancel-text "Close"
        :content
        (let [last-crumb-id (:id (second (:crumbs @state)))
              add-crumb (fn [id text]
                          (swap! state update-in [:crumbs] conj
                                 {:id id :text text
                                  :onClick #(swap! state update-in [:crumbs] (comp vec (partial take 2)))}))]
-         [:div {:style {:position "relative" :minWidth 500}}
+         [:div {:style {:position "relative" :width 720}}
           [:div {:style {:fontSize "150%" :marginBottom "1ex"}}
            [comps/Breadcrumbs {:crumbs (:crumbs @state)}]]
           common/PHI-warning
           [:div {:style {:backgroundColor "white" :padding "1em"}}
            (case last-crumb-id
              :file-import
-             [import-data/Page (select-keys props [:workspace-id :reload :import-type])]
+             [import-data/Page (merge (select-keys props [:workspace-id :reload :import-type]) {:cancel (fn [] (modal/pop-modal))})]
              :workspace-import
              [copy-data-workspaces/Page
               (assoc (select-keys props [:workspace-id :this-realm :reload])
                 :crumbs (drop 2 (:crumbs @state))
                 :add-crumb #(swap! state update-in [:crumbs] conj %)
-                :pop-to-depth #(swap! state update-in [:crumbs] subvec 0 %))]
+                :pop-to-depth #(swap! state update-in [:crumbs] subvec 0 %)
+                :done (fn [] (modal/pop-modal)))]
```

Comment:
Same as above: `(fn [] (modal/pop-modal))` -> `modal/pop-modal`.

Not your code, but in the line above: `... update-in [:single-key] ...` -> `... update :single-key ...`